guide/private-key-management/cloud-backup.md

```diff
@@ -33,14 +33,16 @@ https://www.figma.com/community/file/995256542920917246/BDG---Private-key-manage
    layout = "full-width"
 %}
 
-# Automatic cloud backup
+# Cloud backup
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/wallet/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security. 
+This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security.
```

Comment:
Could add some details here about a Lightning enabled daily spending wallet and how cloud backup can be handy for that.

Reply:
Agree, an automatic cloud backup scheme is likely the best fit for many Lightning wallets as it enables continuous backup of channel states.

Reply:
Agree, automatic cloud backups are likely the best fit for many Lightning wallets as it enables continuous backups of the channel states.

Reply:
Done. I rearranged the order of the paragraph to flow a little better with the addition of the LN content.